pkgs/development/libraries/quarto/default.nix

```
@@ -0,0 +1,56 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
+      mkdir src
+      tar -zxvf ${finalAttrs.src} -C src
+      mkdir -p $out/bin
+      mkdir -p $out/share
+
+      rm src/quarto-${finalAttrs.version}/bin/tools/deno
+      rm src/quarto-${finalAttrs.version}/bin/tools/pandoc
+      rm src/quarto-${finalAttrs.version}/bin/tools/esbuild
+
+      mv src/quarto-${finalAttrs.version}/bin/* $out/bin
+      mv src/quarto-${finalAttrs.version}/share/* $out/share
+
+      ln -s ${deno.outPath}/bin/deno $out/bin/tools/deno
+      ln -s ${pandoc.outPath}/bin/pandoc $out/bin/tools/pandoc
+      ln -s ${esbuild.outPath}/bin/esbuild $out/bin/tools/esbuild
+  '';
+
+  meta = with lib; {
+    description = "Open-source scientific and technical publishing system built on Pandoc. ";
+    longDescription = ''
+        Quarto is an open-source scientific and technical publishing system built on Pandoc.
+        Quarto documents are authored using markdown, an easy to write plain text format.
+    '';
+    homepage = "https://quarto.org/";
+    changelog = "https://github.com/quarto-dev/quarto-cli/releases/tag/v${finalAttrs.version}";
+    license = licenses.gpl2;
```

I see that there are `.wasm` files, and that you haven't replaced all the native executables (such as `dart-sass`).
```suggestion
    license = licenses.gpl2;
    sourceProvenance = with sourceTypes; [ binaryNativeCode binaryBytecode ];
```

---

Solved in [b29176f11449dd4b084d3e59c8bf61856485067f](https://github.com/NixOS/nixpkgs/pull/186697/commits/b29176f11449dd4b084d3e59c8bf61856485067f)